src/entities/appointment.entity.ts

```diff
@@ -14,11 +14,11 @@ export default class Appointment {
   @PrimaryGeneratedColumn()
   id: number;
 
-  @Column()
+  @Column({ default: 'pending' })
   status: string;
 
   @Column()
-  scheduled_at: Date;
+  scheduled_start: Date;
```

Comment:
should this field allow nulls?

Reply:
No

Reply:
then I think you need to add that constraint in the db